scripts/install.sh

```diff
@@ -534,12 +534,43 @@ function ask_to_continue() {
 
 }
 
+# Print information about breaking changes
+function print_breaking_changes() {
+    local versions
+    readonly versions="${1}"
+
+    local changelog
+    readonly changelog="$(echo -e "$(curl --retry 5 --connect-timeout 5 --max-time 30 --retry-delay 0 --retry-max-time 150 -sS https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/CHANGELOG.md)")"
+
+    local is_breaking_change
+    local message
+    message=""
+
+    for version in ${versions}; do
+        # Print changelog for every version
+        is_breaking_change=$(echo -e "${changelog}" | grep -E '^## |^### Breaking|breaking changes' | sed -e '/## \[v'"${version}"'/,/## \[v/!d' | grep -E 'Breaking|breaking' || echo "")
+
+        if [[ -n "${is_breaking_change}" ]]; then
+            if [[ -n "${message}" ]]; then
+                message="${message}, "
+            fi
+            message="${message}v${version}"
+        fi
+    done
+
+    echo ""
+    if [[ -n "${message}" ]]; then
+        echo "The following versions contain breaking changes: ${message}! Please make sure to read the linked Changelog file."
+    fi
+}
+
 # Print changelog link for specific branch
 function print_changelog_link() {
     local branch
     readonly branch="${1}"
 
     echo -e "Changelog:\t\thttps://github.com/SumoLogic/sumologic-otel-collector/blob/${branch}/CHANGELOG.md"
```

Comment:
The only nit is that I think the link to the CHANGELOG.md should point to `main` - it might include fixes to Changelog done after release, and this is what we use when looking for breaking changes anyway.

Reply:
It makes sense, refactored due to that change